frontend/server/src/main/java/org/pytorch/serve/ModelServer.java

```diff
@@ -214,11 +214,6 @@ private void initModelStore() throws InvalidSnapshotException, IOException {
                             if (marMinWorkers > 0 && marMaxWorkers >= marMinWorkers) {
                                 minWorkers = marMinWorkers;
                                 maxWorkers = marMaxWorkers;
-                            } else {
```

Comment:
If someone gives faulty values like 0, -1 etc, shouldn't we have a warning?

Reply:
This is guarded by https://github.com/pytorch/serve/blob/master/frontend/archive/src/main/java/org/pytorch/serve/archive/model/ModelConfig.java#L221

Reply:
Thanks for checking